binlog_writer.go

```diff
@@ -109,5 +109,8 @@ func (b *BinlogWriter) writeEvents(events []DMLEvent) error {
 	if err != nil {
 		return fmt.Errorf("exec query (%d bytes): %v", len(query), err)
 	}
+
+	// lastWrittenBinlogPosition = events[len(events)-1].BinlogPosition()
```

Comment:
This is temporary. Once this merges, I'll change this line in #60 such that we send this position to the StateTracker.